src/config/config_service.py

```diff
@@ -46,14 +49,22 @@ def _preprocess_incoming_config(config):
     config['name'] = name.strip()
 
 
+def _create_archive_filename(filename):
+    current_datetime = datetime.now()
+    formatted_datetime = current_datetime.strftime('%Y%m%d%H%M%S')
+    return f"{formatted_datetime}_{filename}.deleted"
```

Comment:
Should we use `.` instead of `_` for date separator? With underscore it looks a bit like a part of an original name (at least for me, because I use `_` for files naming, e.g. `bash_formatting.json`)

Reply:
yeah sure that can be a `.`